src/backend/joanie/core/utils/issuers.py

```diff
@@ -0,0 +1,35 @@
+"""Utils that can be useful throughout Joanie's core app for document issuers"""
+from pathlib import Path
+
+from django.template import Context
+from django.template.engine import Engine
+from django.template.loader import get_template
+
+from weasyprint import CSS, HTML
+from weasyprint.text.fonts import FontConfiguration
+
+
+def generate_document(path: str, instance) -> bytes:
+    """
+    Generate the document with a given path and the instance
+    of an object we would like to retrieve data from.
+    Note:
+        - The path will help us in finding the correct .html and
+        .css file for the document in the app directory.
+        Make sure that those files exist in the 'templates' folder.
+    """
+    html_template_path = Path(f"issuers/{path}.html")
+    css_template_name = Path(f"issuers/{path}.css")
+
+    context_query_dict = instance.get_document_context()
+
+    doc_html = HTML(
+        string=Engine.get_default()
+        .get_template(html_template_path)
+        .render(Context(context_query_dict))
+    )
+    css = CSS(
+        string=get_template(css_template_name).render(context_query_dict),
+        font_config=FontConfiguration(),
+    )
+    return doc_html.write_pdf(stylesheets=[css], zoom=1, uncompressed_pdf=True)
```

Comment:
If our tests does not break when compression is enabled, keep turned on the pdf compression.

```suggestion
    return doc_html.write_pdf(stylesheets=[css], zoom=1)
```